### PR TITLE
Run schedulers workstream integ tests only on alinux2

### PR DESCRIPTION
**Pull request by jdeamicis**
### Description of changes
* Change the integration tests setup for schedulers.yaml (only used for internal testing purposes).

### Checklist
- [x] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [x] Check all commits' messages are clear, describing what and why vs how.
- [ ] ~Make sure **to have added unit tests or integration tests** to cover the new/modified code.~
- [ ] ~Check if documentation is impacted by this change.~

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integration-tests/configs/schedulers.yaml [hunk 1]
```diff
@@ -20,24 +20,24 @@ test-suites:
...
           schedulers: ["slurm"]
   scaling:
     test_mpi.py::test_mpi:  # TODO: move outside of the scaling dir
       dimensions:
         - regions: ["ap-east-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: {{ common.OSS_COMMERCIAL_X86 }}
+          oss: ["alinux2"]
           schedulers: ["slurm"]
         - regions: ["eu-north-1"]
```
Comment (NSsirena) [inline tests/integration-tests/configs/schedulers.yaml]: This is the same test above just run on ARM.
I think for the internal tests we can skip it.

File: tests/integration-tests/configs/schedulers.yaml [hunk 2]
```diff
@@ -60,27 +60,27 @@ test-suites:
...
+          oss: ["alinux2"]
           schedulers: ["slurm"]
     test_slurm.py::test_slurm_pmix:  # TODO: include in main test_slurm to reduce number of created clusters
       dimensions:
         - regions: ["ap-south-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: {{ common.OSS_COMMERCIAL_X86 }}
+          oss: ["alinux2"]
           schedulers: ["slurm"]
         - regions: ["ap-northeast-1"]
```
Comment (NSsirena) [inline tests/integration-tests/configs/schedulers.yaml]: Also this is the same test run on ARM.
We can skip it in our internal test suite

File: tests/integration-tests/configs/schedulers.yaml [hunk 3]
```diff
@@ -60,27 +60,27 @@ test-suites:
...
+          oss: ["alinux2"]
           schedulers: ["slurm"]
     test_slurm.py::test_slurm_scaling:
       dimensions:
         - regions: ["us-west-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: {{ common.OSS_ONE_PER_DISTRO }}
+          oss: ["alinux2"]
           schedulers: ["slurm"]
         - regions: ["use2-az2"]  # do not move, unless instance type support is moved as well
```
Comment (NSsirena) [inline tests/integration-tests/configs/schedulers.yaml:69]: Also this, is the same test run on a different(?)  instance type

File: tests/integration-tests/configs/schedulers.yaml [hunk 4]
```diff
@@ -198,15 +198,14 @@ test-suites:
       dimensions:
         - regions: ["eu-west-2"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: {{ common.OSS_COMMERCIAL_X86 }}
+          oss: ["alinux2"]
           schedulers: ["slurm"]
         - regions: ["ap-northeast-1"]
```
Comment (NSsirena) [inline tests/integration-tests/configs/schedulers.yaml]: Same as above

File: tests/integration-tests/configs/schedulers.yaml [hunk 5]
```diff
@@ -20,24 +20,24 @@ test-suites:
...
         - regions: ["us-east-2"]
           instances: ["c5.18xlarge"]
-          oss: ["centos7"]
+          oss: ["alinux2"]
           schedulers: ["slurm"]
   scaling:
     test_mpi.py::test_mpi:  # TODO: move outside of the scaling dir
       dimensions:
         - regions: ["ap-east-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
```
Comment (jdeamicis) [inline tests/integration-tests/configs/schedulers.yaml:29]: test comment

File: tests/integration-tests/configs/schedulers.yaml (hunk 1 above)
Comment (jdeamicis) [reply tests/integration-tests/configs/schedulers.yaml]: Done (I removed the one for ARM).

File: tests/integration-tests/configs/schedulers.yaml (hunk 2 above)
Comment (jdeamicis) [reply tests/integration-tests/configs/schedulers.yaml]: Done (I removed the one for ARM).

File: tests/integration-tests/configs/schedulers.yaml (hunk 3 above)
Comment (jdeamicis) [reply tests/integration-tests/configs/schedulers.yaml:69]: Done (I removed the one for ARM).

File: tests/integration-tests/configs/schedulers.yaml (hunk 4 above)
Comment (jdeamicis) [reply tests/integration-tests/configs/schedulers.yaml]: Done (I removed the one for ARM).

**Review by NSsirena — APPROVED** [review]

**Pull request merged**